INSTALL.md

```diff
@@ -178,16 +178,33 @@ make install
 
 In addition to meeting the [requirements](#requirements), you need a version of Visual Studio (the [community edition](https://www.visualstudio.com/vs/visual-studio-express/) is enough) and a CMake version that's compatible with the Visual Studio version you're using.
 
-You must also ensure that the msvc versions of dependencies you're using are placed in the correct folders.
+With the dependencies you have two options:
```

Comment:
Just nitpicking here so don't take this the wrong way. I'd propose replacing "With" with "In regards to" or "Regarding".

Reply:
Done!